src/met-atlas-viewer.js

```diff
@@ -924,22 +921,22 @@ function MetAtlasViewer(targetElement) {
       };
       return await setData(filteredData);
     }
-  }
+  };
 
   /**
    * Sets the distance to show node labels.
    *
    * @param {number} newDistance The new label rendering distance
    */
-  function setLabelDistance(newDistance) {
-    labelDistance = newDistance
+  const setLabelDistance = (newDistance) => {
+    labelDistance = newDistance;
     animationId = requestAnimationFrame(render);
-  }
+  };
 
   /**
    * Returns a list of all nodes within the given `distance` from the camera.
    */
-  function getNodesWithin(distance) {
+  const getNodesWithin = (distance) => {
     let nodes = [];
 
     let testCamera = new PerspectiveCamera(fieldOfView, aspect, near, distance);
```

Comment:
Shouldn't we be able to use helper function `moveCamera` here as well?

Reply:
Makes sense! I had to rewrite the `moveCamera` function a bit to accommodate the new params. Addressed in: https://github.com/MetabolicAtlas/3d-network-viewer/pull/51/commits/f7661ce692bb7b6b94229e36185a021bd2516ac5